docs/PUBLISHING.md

```diff
@@ -76,7 +85,7 @@ Make sure you set the following environment variables:
 Run the following command:
 
 ```sh
-make publish-bintray-debian RELEASE_TYPE=<production|snapshot>
+make publish-bintray-debian
```

Comment:
...and when you do that, I guess you ought to provide instructions on how someone could subscribe to the snapshot Debian feed ;-)